import/controls/permissionmanager.h

```diff
@@ -0,0 +1,30 @@
+#ifndef PERMISSIONMANAGER_H
+#define PERMISSIONMANAGER_H
+
+#include <QObject>
+
+class PermissionManager : public QObject
+{
+    Q_OBJECT
+public:
+    PermissionManager(QObject *parent = nullptr);
+
+    // permission cases: 'unknown', 'allow', 'deny', 'prompt'
```

Comment:
These should be declared as enums.

Reply:
Indeed. Better to store this value as an integer instead of a string constant. The same is about the `embedlite-components` part - I think it's possible to send the integer value instead of the string.